apps/file_publisher/lib/file_publisher/file_publisher.ex

```diff
@@ -12,11 +12,19 @@ defmodule FilePublisher do
     GenServer.start_link(__MODULE__, file_path, name: __MODULE__)
   end
 
+  def start_link(name, file_path) do
+    GenServer.start_link(__MODULE__, file_path, name: name)
+  end
+
   @impl Core.PublisherContract
   def handle_message(message) do
     GenServer.cast(__MODULE__, {:replication_message, message})
```

Comment:
Maybe could call `handle_message(__MODULE__, message)` now? If we even want to keep it.

Reply:
I will change i to `handle_message(name, message)`.